util/users.py

```diff
@@ -55,3 +55,21 @@ async def delete_user(identifier: user_models.UserIdentifier) -> None:
     if response.deleted_count == 0:
         detail = "User not found and could not be deleted"
         raise exceptions.UserNotFoundException(detail=detail)
+
+
+async def validate_user_info(
+        validate: user_models.ValidateUserForm) -> str:
```

Comment:
Consider changing the return type (and type hints) to be `user_models.ValidateUserResponse` (post `login`-refactor) for convenience of the caller.